app/services/paypal_service/logger.rb

```diff
@@ -2,29 +2,28 @@ module PaypalService
   class Logger
 
     def info(msg)
-      #TODO: Fix logging totally
-      Rails.logger.warn "[Paypal] #{msg}"
+      Rails.logger.info ({ paypal: msg }.to_json)
```

Comment:
Because this is used for the requests and responses from paypal, and they contain json, not a string message.

We could of course create more consistent schema with e.g.:
```
 { paypal: { type: :info, message: nil, content: <request hash> }
 { paypal: { type: :error, message: "error message", content: nil }
```